docs/ui/components/Callout/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { css } from '@emotion/react';
+import { borderRadius, iconSize, theme, typography } from '@expo/styleguide';
+import React, { ComponentType, PropsWithChildren } from 'react';
+
+import { IconProps, ErrorIcon, InfoIcon, WarningIcon } from '~/ui/foundations/icons';
+
+type CalloutType = 'info' | 'warning' | 'error';
+
+type CalloutProps = PropsWithChildren<{
+  type?: CalloutType;
+  icon?: ComponentType<IconProps> | string;
+}>;
+
+export const Callout = ({ type = 'info', icon, children, ...rest }: CalloutProps) => {
+  const Icon = icon || getCalloutIcon(type);
+  return (
+    <div css={[containerStyle, getCalloutColor(type)]} {...rest}>
+      <i css={iconStyle}>{typeof icon === 'string' ? icon : <Icon size={iconSize.small} />}</i>
+      <div css={contentStyle}>{children}</div>
+    </div>
+  );
+};
+
+function getCalloutColor(type: CalloutType) {
+  switch (type) {
+    case 'warning':
+      return warningColorStyle;
+    case 'error':
+      return errorColorStyle;
+    default:
+      return null;
+  }
+}
+
+function getCalloutIcon(type: CalloutType) {
+  switch (type) {
+    case 'warning':
+      return WarningIcon;
+    case 'error':
+      return ErrorIcon;
+    default:
+      return InfoIcon;
+  }
+}
+
+const containerStyle = css({
+  backgroundColor: theme.background.secondary,
+  border: `1px solid ${theme.border.default}`,
+  borderRadius: borderRadius.medium,
+  display: 'flex',
+  padding: '1rem',
+});
+
+const iconStyle = css({
+  fontStyle: 'normal',
+  marginRight: '0.5rem',
+  userSelect: 'none',
+});
+
+const contentStyle = css({
+  ...typography.body.paragraph,
+  color: theme.text.default,
```

Comment:
Hate to do this, but we can't just use a `<P>` bc markdown adds paragraphs and we can't nest paragraphs.

Reply:
In hindsight, they are actually correct in adding the `P` tag. If a blockquote is written like:

```
> 🎨 **Look how cool that is!***
>
> Bla bla, it's because it's Expo, bla bla
```

We want to render these two separate blocks with a margin, the `P` tag takes care of that. So I think they are right, but it's just super annoying to also have to support it 🙈  (luckily we do now)